.github/workflows/tests.yml

```
@@ -11,14 +11,14 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
-        os: [ubuntu-20.04, macos-13, macos-14, windows-latest]
+        os: [ubuntu-latest, macos-13, macos-14, windows-latest]
 
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)